client/package.json

```diff
@@ -18,7 +18,8 @@
     "ci:cypress:staging": "CYPRESS_GRAPHQL_ENDPOINT=https://rtop-server.herokuapp.com/v1alpha1/graphql cypress run --record --browser chrome -c baseUrl=\"https://staging.sketch.sh\",defaultCommandTimeout=20000",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "cy:run:record": "cypress run --record --browser chrome"
+    "cy:run:record": "cypress run --record --browser chrome",
+    "refmt": "yarn bsrefmt --in-place $(find src shared src_editor src_analyzer src_embed src_worker -type f \\( -name *.re -o -name *.rei \\))"
```

Comment:
@feihong I added some command in https://github.com/Sketch-sh/sketch-sh/pull/333, where I was including opam to fix integration with vscode.

Now there would be two ways to format the code, do you think we should use `bsrefmt` or plain `refmt`?

Reply:
I'm not clear on the difference between bsrefmt and refmt, I thought one was an alias for the other

Reply:
Ok, let's use `bsrefmt` (less chances for breakages due to different versions).

Please remove the `format` and `format-check` commands in makefile:

https://github.com/Sketch-sh/sketch-sh/blob/329566784064ca216eec05b98d510568f7db52f4/client/Makefile#L35-L41

Reply:
@jchavarri It seems that [bsrefmt is recommended over refmt](https://reasonml.chat/t/bsrefmt-or-refmt-with-vscode-reason-vscode-1-5-2/1470/4)

> BuckleScript occasionally copies over newer refmt version whenever there were important updates. For consistency reasons, it’s usually recommended to go with the bsrefmt over any other globally installed refmt version (makes it easier to have reproducible results on CI and your coworkers machine).

Reply:
Removed unused format commands in https://github.com/Sketch-sh/sketch-sh/pull/323/commits/11dda1d2dbc781af8ef3480bdd3b1c452a7e2d69